packages/ui-portal/src/Portal/index.tsx

```diff
@@ -55,6 +56,10 @@ class Portal extends Component<PortalProps, PortalState> {
   constructor(props: PortalProps) {
     super(props)
 
+    if (!canUseDOM) {
+      return
+    }
+
```

Comment:
It probably would be nice to throw some kind of warning here so they know why Portals (and Modals, Popovers, Tooltips etc...) don't render. Also might be worth adding info about it in the SSR docs.

Reply:
I added it to the guide: 
https://github.com/instructure/instructure-ui/pull/879/files#diff-663a30594b4684ce916d76763d1f44e4aeeae00840593c32ad246483435af7a3R21-R24

Reply:
Also added the warning to the render method instead of the constructor.